main31.js

```diff
@@ -1,73 +1,84 @@
-import { CLOSE_ICON, MESSAGE_ICON, TICK_ICON, style, setColorsFromThemeName } from "./assets15.js";
+import {
+  CLOSE_ICON,
+  MESSAGE_ICON,
+  TICK_ICON,
+  style,
+  setColorsFromThemeName,
+} from "./assets15.js";
 
 let loadInterval;
 var chatbotID = "";
-//const BASE_URL = "http://localhost:5001/dev/";
-const BASE_URL = "https://sswj8m0la3.execute-api.af-south-1.amazonaws.com/dev/";
+const BASE_URL = "http://localhost:5001/dev/";
+//const BASE_URL = "https://sswj8m0la3.execute-api.af-south-1.amazonaws.com/dev/";
```

Comment:
Please switch which base url is being used, as we want this to be the live version